optimizer/quickstep/query_optimizer/tests/CMakeLists.txt

```diff
@@ -16,9 +16,9 @@
 # under the License.
 
 #add_subdirectory(execution_generator)
-add_subdirectory(logical_generator)
```

Comment:
Did some tests start failing after the changes you made?

Reply:
Yes because the Hustle resolver doesn't support all the queries in the tests. I'll work this out in a future PR.